R/stao/commercial_crew_submission_114.R

```diff
@@ -0,0 +1,112 @@
+################
+#Issue #114: Commercial Crew Member Data
+#Data Submission
+#February 2018
+#Sophia Tao
+################
+
+
+
+# load libraries
+library(arcticdatautils)
+library(dataone)
+library(EML)
+library(XML)
+library(digest)
+library(shiny)
+library(rhandsontable)
+
+# set environment 
+cn <- CNode('PROD')
+mn <- getMNode(cn,'urn:node:KNB')
+# set authentication token
+
+
+# publish file
+# formatted <- publish_object(mn, '/home/stao/my-sasap/114_commercial_crew/Commercial_Crew_data_2012-2017_formatted.csv', format_id = 'text/csv')
+formatted <- "urn:uuid:f8a813e8-4937-40ed-9312-ddb177a6469b"
+
+
+
+# edit EMl
+eml_path <- '/home/stao/my-sasap/114_commercial_crew/CommercialCrewEML.xml'
+eml <- read_eml(eml_path)
+
+# add SASAP project info
+source('~/sasap-data/data-submission/Helpers/SasapProjectCreator.R')
+eml@dataset@project <- sasap_project() 
+
+# add intellectual rights
+eml@dataset@intellectualRights <- new('intellectualRights',
+                                      .Data = "CFEC retains intellectual property rights to data collected by or for CFEC. Any dissemination of the data must credit CFEC as the source, with a disclaimer that exonerates the department for errors or deficiencies in reproduction, subsequent analysis, or interpretation. Please see http://www.adfg.alaska.gov/index.cfm?adfg=home.copyright for further information.")
+
+# change abstract
```

Comment:
Tip: you can create "sections" in RStudio if you add `------` or `#####` to your comments. See here for more info: https://support.rstudio.com/hc/en-us/articles/200484568-Code-Folding-and-Sections